freqtrade/exchange/binance.py

```diff
@@ -217,3 +224,58 @@ def load_leverage_tiers(self) -> Dict[str, List[Dict]]:
                     raise OperationalException(e) from e
         else:
             return {}
+
+
+
+    @retrier
+    @cached(cache=TTLCache(maxsize=100, ttl=10), lock=Lock())
```

Comment:
you should take your pick, really, when it comes to caching.

actual caching is applied below (in the `get_spot_pair_delist_time()`.
applying caching twice can be dangerous here - as then you risk having the TTL applied twice in unfortunate cases.

That decorator is also something we're not using anywhere else - as it won't allow you to work around the cache and force a refresh - which can be necessary in some instances.